entries/normalizer.xml

```diff
@@ -38,21 +41,67 @@ $( "#myform" ).validate( {
 <input type="submit" value="Validate!">
 ]]></html>
 	</example>
-    <example>
-        <desc>Makes "url" required and use a normalizer to append 'http://', if not present, to the value of the "url" element before validating</desc>
+	<example>
+		<desc>Makes "url" required and use a normalizer to append 'http://', if not present, to the value of the "url" element before validating</desc>
+		<code><![CDATA[
+$( "#myform" ).validate( {
+  rules: {
+    url_input: {
+      required: true,
+      url: true,
+      normalizer: function( value ) {
+        var url = value;
+
+        // Check if it doesn't start with http:// or https:// or ftp://
+        if ( url && url.substr( 0, 7 ) !== "http://"
+            && url.substr( 0, 8 ) !== "https://"
+            && url.substr( 0, 6 ) !== "ftp://" ) {
+          // then prefix with http://
+          url = "http://" + url;
+        }
+
+        // Return the new url
+        return url;
+      }
+    }
+  }
+} );
+]]></code>
+<html><![CDATA[
+<label for="url_input">url: </label>
+<input class="left" id="url_input" name="url_input">
+<br/>
+<input type="submit" value="Validate!">
+]]></html>
+	</example>
+
+	<example>
+		<desc>Using a global normalizer in conjunction with a local one</desc>
```

Comment:
AFAIK, we never added the version to any of the available API, nor it was added before.
If you want I can add it, but that will be inconsistent with the rest of the documentation.